src/views/admin/UserInfo.tsx

```diff
@@ -57,9 +58,21 @@ const UserInfo = () => {
 
       <Modal
         open={openModal}
-        className="w-[400px] mx-auto flex justify-center items-center"
+        className=" w-80 mx-auto flex justify-center items-center bg-red-100 absolute p-4 max-w-2xl max-h-96 my-auto rounded-lg"
       >
         <form onSubmit={userForm.handleSubmit}>
+          <div className="grid">
+            <CustomButton
+              onClick={() => {
+                setOpenModal(false);
+              }}
+              variant="contained"
+              className="rounded-full max-w-0 justify-self-end"
+            >
+              &#x2715;
```

Comment:
What does the line 72 do

Reply:
That is the character code i used to write the X on the close button